week-5-practise/js/main.js

```diff
@@ -0,0 +1,103 @@
+/*### part 1
+
+whhen clicking **blue** it should change:
+        - **Jumbotron** background color to `#588fbd`
+        - **Donate a bike** button background color to `#ffa500`
+        - **Volunteer** button background color to `black` and text color to `white`*/
+
+var blueTheme = document.querySelector("#blueBtn");
+var myContainer = document.querySelector(".jumbotron");
+var myButton = document.querySelectorAll(".btn-lrg");
+var volunteer = myButton[1];
+var donateBike = myButton[0];
+blueTheme.addEventListener("click", blueBackground);
+function blueBackground() {
+  myContainer.style.backgroundColor = "#588fbd";
+  donateBike.style.backgroundColor = "#ffa500";
+  volunteer.style.backgroundColor = "black";
+  volunteer.style.color = "white";
+}
+
+/*When clicking **orange** it should change:
+        - **Jumbotron** background color to `#f0ad4e`
+        - **Donate a bike** button background color to `#5751fd`
+        - **Volunteer** button background color to `#31b0d5` and text color to `white`*/
+
+var orangeTheme = document.querySelector("#orangeBtn");
+var myContainer = document.querySelector(".jumbotron");
+var myButton = document.querySelectorAll(".btn-lrg");
```

Comment:
You could have saved yourself some of these queries and assignments here as you've already got them above.

Reply:
how? I wanted to but dint know how. and without it, it didnt work

Reply:
You could for example delete lines 28-30 and nothing should change as those 3 variables were declared in lines 10-12.